pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -460,6 +470,33 @@ export default class WindowsIntegrationManager implements IntegrationManager {
     }
   }
 
+  /**
```

Comment:
The indent here (and next function) is odd.

Obviously I'll probably merge it as-is anyway :p